core/merge.js

```diff
@@ -275,6 +275,7 @@ class Merge {
           }
           return this.pouch.put(file)
         } else {
+          // TODO: nécessité de sauvegarder quand même le document ??
```

Comment:
You have some TODOs in french. Is it normal?

Reply:
Nope! Good catch. I'll rewrite it in English.